.gitignore

```diff
@@ -1,2 +1,3 @@
+.idea
```

Comment:
```suggestion
```

Files that are not relevant to the repository should be ignored locally. See https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally

> Patterns which are specific to a particular repository but which do not need to be shared with other related repositories (**e.g., auxiliary files that live inside the repository but are specific to one user's workflow**) should go into the $GIT_DIR/info/exclude file.
> 
> The .git/info/exclude file has the same format as any .gitignore file. **Another option is to set core.excludesFile to the name of a file containing global patterns.**

Reply:
If this is a generic standard here, maybe we should add it to the developers manifest? Quick search yielded no results: https://github.com/BrandEmbassy/developers-manifest/search?q=gitignore

Reply:
I'm not really sure whether it's a generic standard in NICE, but I definitely view it as a best practice, especially in open source development (which this library technically is).

It's a good idea to propose to add it into our Developers Manifest, will you create a PR?